flow/core/experiment.py

```diff
@@ -54,12 +53,18 @@ class can generate csv files from emission files produced by sumo. These
     ----------
     env : flow.envs.Env
         the environment object the simulator will run
+    custom_callables : [str, lambda]
+        List of strings and lambda functions corresponding to some information we want
+        to extract from the environment. The lambda will be called at each step to extract
+        information from the env and it will be stored in a dict keyed by the str.
```

Comment:
maybe add something that they should return a numerical quantity. i think this script breaks if the callable returns None